docs/sources/explore/correlations-editor-in-explore.md

```diff
@@ -0,0 +1,118 @@
+1. Explore is now in Correlations Editor mode indicated by a blue border and top bar. You can exit Correlations Editor using the Exit button in the top bar.
+1. The visualization is enriched with links allowing to create new correlations. Links are attached to the data that you can use to build a new query:
+   - Logs: links are displayed next to field values inside log details for each log row
+   - Table: every table cell is a link
+1. Click on a link to begin adding new correlation. Links are associated with a field that will be used as [a results field of a correlation]({{< relref "../administration/correlations/correlation-configuration" >}}). This means the correlation link will be created on the link you select once the correlation is saved.
+1. Explore opens a split view. Use the right pane to setup [the target query source of the correlation]({{< relref "../administration/correlations/correlation-configuration#target-query" >}}).
+1. Build target query using [variables syntax]({{< relref "../dashboards/variables/variable-syntax" >}}) with variables from the list provided at the top of the pane. The list contains sample values from the selected data row.
+1. Provide optional label and description. Label is used as the name of the link inside the visualization. It can contain variables.
+1. Click "Save" button in the top bar to save the correlation and exit Correlations Editor mode.
+1. The link used to create the correlation will be replaced with a data link in each row. The variables will be dynamically replaced with values from a selected row.
+
+## Transformations
+
+Transformations allow us to extract values that exist in a field with other data. A prime example is getting one portion of a log line out to use in a correlation. For more details on transformations in correlations, see the [correlations documentation]({{< relref "../administration/correlations/correlation-configuration/#correlation-transformations" >}}).
+
+After clicking one of the generated links in the editor mode, you can add transformations by clicking the "Add transformation" button in the "Transformations" dropdown. This will pop up a modal, and you can use a transformation in your correlation with the following steps:
+
+1. First, select a field that the transformation will be applied. This should be a field where we only want to use sections of the field, such as a log line. Once selected, the value of that field will show up to assist building the transformation.
+1. Select the type of the transformation. See the [correlations documentation]({{< relref "../administration/correlations/correlation-configuration/#correlation-transformations" >}}) for the options and relevant settings.
+1. Based on what's selected, you may immediately see one or more variables populate, or you may need to fill in more fields
```

Comment:
```suggestion
1. Based on what's selected, you may immediately see one or more variables populate, or you may need to fill in more options
```

There's a bit of ambiguity "form field" vs "data frame field" (step 1). Maybe "options" would be more clear here?